test/e2e/containers/containers.go

```diff
@@ -272,7 +272,7 @@ func (m *Manager) RunNodeResource(chainId string, containerName, valCondifDir st
 		Entrypoint: []string{
 			"sh",
 			"-c",
-			"babylond start " + FlagHome,
+			"babylond start " + FlagHome + " --log_level trace --trace",
```

Comment:
if we want to use trace log level should be modified at https://github.com/babylonlabs-io/babylon/blob/ee744545e9e6369cd1544f8f6d1bb4f64b634643/test/e2e/initialization/node.go#L358

But in my opinion trace is too much :sweat_smile:

Reply:
This seems to be debug leftover, removed